.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: ci
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        go-version: [1.14.x, 1.15.x]
```

Comment:
let's just test on 1.15.x, because once 1.16 is out we want to drop 1.15 and older versions to be able to use `go:embed` for the assets